From: Fedora/Cygwin
Subject: Enable SSH_SOURCE_BASHRC

Sources ~/.bashrc when bash detects it is being run via SSH.
Checks for SSH_CLIENT or SSH2_CLIENT in the environment.

--- a/config-top.h
+++ b/config-top.h
@@ -103,7 +103,7 @@ typedef void *sh_parser_state_t;
 /* Define this if you want bash to try to check whether it's being run by
    sshd and source the .bashrc if so (like the rshd behavior).  This checks
    for the presence of SSH_CLIENT or SSH2_CLIENT in the initial environment,
    which can be fooled under certain not-uncommon circumstances. */
-/* #define SSH_SOURCE_BASHRC */
+#define SSH_SOURCE_BASHRC
 
 /* Define if you want the case-toggling operators (~[~]) and the
    `capcase' variable attribute (declare -c). */
